### DISCUSSION TITLE
List down limitations of this JSON RPC relay

### DISCUSSION THREAD

**Discussion by kabaluyot** [hiero-ledger/hiero-json-rpc-relay#1202] [Q&A] [answered]
### Problem

The docs explicitly mentioned that this JSON RPC replay got limitations but there is no list of what features are supported or not. Currently manually checking for stuff is supported or not or just going "try it out and find out"

### Solution

Include list of features in README and known limitations

**Comment by Nana-EC** [comment, marked as answer]
Hey @kabaluyot, thanks for highlighting this.
Is documentation around the supported endpoints what you're looking for - something like https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/docs/rpc-api.md#endpoint-table ?
If not which features or concepts do you think might be missing so we can improve documentation?

**Comment by kabaluyot** [comment]
Yes -- thanks for this

**Comment by Nana-EC** [comment]
NP. 
Turning this into a discussion as it's more suited to a Q&A

**Comment by Nana-EC** [comment]
Closing this off.
Limitations may have referred to early versions of the relay.
At this point any gaps are bugs or items not supported.
Docs were improved - noting https://docs.hedera.com/hedera/core-concepts/smart-contracts/json-rpc-relay and main README
